src/views/Change.vue

```diff
@@ -0,0 +1,294 @@
+    if (!this.isAuthenticated) return
+
+    // do not proceed if FF is disabled
+    // bypass this when Jest is running as FF are not fetched
+    if (!this.isJestRunning && !getFeatureFlag('change-ui-enabled')) {
+      window.alert('Change filings are not available at the moment. Please check again later.')
+      this.$root.$emit('go-to-dashboard')
+      return
+    }
+
+    // try to fetch data
+    try {
+      const firmSnapshot = await this.fetchFirmSnapshot()
+
+      if (this.changeId) {
+        // store the filing ID
+        this.setFilingId(this.changeId)
+
+        // fetch draft change filing to resume
+        const changeFiling = await this.fetchFilingById(this.changeId)
+
+        // do not proceed if this isn't a Change filing
+        if (!changeFiling.changeFirm) {
+          throw new Error('Invalid Change filing')
+        }
+
+        // do not proceed if this isn't a DRAFT filing
+        if (changeFiling.header.status !== FilingStatus.DRAFT) {
+          throw new Error('Invalid Change status')
+        }
+
+        // parse change filing and original business snapshot into store
+        await this.parseChangeFirm(changeFiling, firmSnapshot)
+      } else {
+        // parse business data into store
+        await this.parseFirmSnapshot(firmSnapshot)
+      }
+
+      // Set the resources
+      this.setResource(this.changeFirmResources)
+
+      // initialize Fee Summary data
+      this.setFilingData(this.changeFirmResources.filingData)
+
+      // update the current fees for the Filing
+      this.setCurrentFees(
+        await this.fetchFilingFees(
+          FilingCodes.CHANGE_FIRM, this.getEntityType
+        ).catch(() => cloneDeep(EmptyFees))
+      )
+
+      // fetches the fee prices to display in the text
+      this.setFeePrices(
+        await this.fetchFilingFees(FilingCodes.CHANGE_FIRM, this.getEntityType
+        ).catch(() => cloneDeep(EmptyFees))
+      )
+
+      // tell App that we're finished loading
+      this.emitHaveData()
+    } catch (err) {
+      console.log(err) // eslint-disable-line no-console
+      this.emitFetchError(err)
+    }
+
+    // now that all data is loaded, wait for things to stabilize and reset flag
+    Vue.nextTick(() => this.setHaveUnsavedChanges(false))
+  }
+
+  /** Fetches the business snapshot. */
+  private async fetchFirmSnapshot (): Promise<FirmSnapshotIF> {
+    const items = await Promise.all([
+      this.fetchBusinessInfo(),
+      this.fetchAuthInfo(),
+      this.fetchOrgPersons()
+    ])
+
+    if (items.length !== 3) throw new Error('Failed to fetch business snapshot')
+
+    return {
+      businessInfo: items[0],
+      authInfo: items[1],
+      orgPersons: items[2]
+    }
+  }
+
+  /** Emits Fetch Error event. */
+  @Emit('fetchError')
+  private emitFetchError (message: string = ''): void { }
+
+  /** Emits Have Data event. */
+  @Emit('haveData')
+  private emitHaveData (haveData: Boolean = true): void { }
```

Comment:
`Boolean` is the class. `boolean` is the type. This should be the latter.

Reply:
typo, good catch

Reply:
Updated!